edml-1.0.0.json

```diff
@@ -0,0 +1,215 @@
+          "properties": {
+            "destinationTable": {
+              "$ref": "#/definitions/destinationTable"
+            },
+            "mapping": {
+              "$ref": "#/definitions/mappingDefinition",
+              "description": "Defines the mapping for the columns of the new table. If the DynamoDB list contains non objects like strings here directly a toString mapping can be applied. Otherwise, if the list contains objects, you can navigate through its properties using fields."
+            },
+            "description": {
+              "$ref": "#/definitions/description"
+            }
+          },
+          "required": [
+            "mapping"
+          ],
+          "additionalProperties": false
+        }
+      },
+      "additionalProperties": false
+    },
+    "description": {
+      "type": "string",
+      "title": "Optional description",
+      "description": "Using this property you can provide documentation."
+    },
+    "required": {
+      "type": "boolean",
+      "default": false,
+      "description": "If this property is set to true, queries will abort if the given mapping can't be found a single document. Else, the query will continue and set this cell to a default value. By default this is property is set to false."
+    },
+    "key": {
+      "type": "string",
+      "enum": [
+        "local",
+        "global"
+      ],
+      "description": "Setting this property marks a column as key.\n\n Key columns are used as foreign keys from ToTableMappings for nested lists. \n\nA key can either be local or global. A global key must be unique over all rows of the table. A local key must only be unique in the scope of the nested list."
+    }
+  },
+  "type": "object",
+  "title": "The Root Schema - Represents the mapping from a DynamoDB table to an Exasol Virtual Schema table",
+  "description": "The structure of this mapping definition follows the structure of the data in the DynamoDB. This object represents a DynamoDB table. Its properties are modeled in the fields property of this object. Only properties of the DynamoDB table that shall be mapped are modeled. \n\n Each of this objects must be stored in a single .json file an get uploaded to an directory in bucketfs.",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "source",
+    "destinationTable",
+    "mapping"
+  ],
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "description": "Reference to the mapping language schema. Setting this is required so that IDEs can help you when writing your definition.",
```

Comment:
```suggestion
      "description": "Reference to the mapping language schema. Setting this is required so that IDEs can assist you while writing your definition.",
```